api/v1alpha1/profile_types.go

```diff
@@ -28,17 +37,23 @@ type ProfileDefinitionSpec struct {
 	// Description is some text to allow a user to identify what this profile installs.
 	Description string `json:"description,omitempty"`
 	// Artifacts is a list of Profile artifacts
-	// can be one of HelmChart, TODO
 	Artifacts []Artifact `json:"artifacts,omitempty"`
 }
 
+// Artifact defines a bundled resource of the components for this profile.
 type Artifact struct {
 	// Name is the name of the Artifact
 	Name string `json:"name,omitempty"`
 	// Path is the local path to the Artifact in the Profile repo
 	Path string `json:"path,omitempty"`
-	// Kind is the kind of artifact: HelmChart or Kustomize
+	// Kind is the kind of artifact: HelmChartLocal or Kustomize
 	Kind string `json:"kind,omitempty"`
+	// HelmURL is the URL of the Helm repository containing a Helm chart and possible values
```

Comment:
I wonder if this isn't extractable into a separate struct?

The tuple of (`chart repository`, `chart`, `version`) feels like a value type.

Reply:
Good idea. The separation would be nice indeed.

Reply:
This is done.